RobotAppEyes/RobotAppEyes.py

```diff
@@ -160,43 +194,30 @@ def open_eyes_session(self,
             intheight = int(height)
             eyes.open(driver, appname, testname, {'width': intwidth, 'height': intheight})
 
-
-    def check_eyes_window(self, name, force_full_page_screenshot=False,
-                          includeEyesLog=False, httpDebugLog=False):
+    def check_eyes_window(self, name, includeEyesLog=False, httpDebugLog=False):
```

Comment:
Why not allow the `force_full_page_screenshot` here too?

Reply:
Because you have to use first the open_eyes_session. So you can make a screenshot with the check_eyes_window function. However, the new functions uses also the check_eyes_window and then you have to set it more times in your code. That's why I removed it here and put it only in the  open_eyes_session.

Reply:
I understand that but now you make a breaking change to how this function works (old code that uses the argument no longer works)